### PR TITLE
Restrict synthetics/browser from fleet

### PR DESCRIPTION
**Pull request by andrewvc**
We do not yet have a synthetics fleet image available, so, for now, users should not be allowed to create synthetics inputs in fleet.

@michalpristas can you verify that this is the right place for this file to exist? Once we have a special big docker image with synthetics deps we can conditionally remove this restriction in that special image (which has access to real browsers and graphics libs).

Fixes https://github.com/elastic/beats/issues/25775
See also https://github.com/elastic/beats/issues/22932#issuecomment-841301435

## Checklist

~- [ ] My code follows the style guidelines of this project~
- [x] I have commented my code, particularly in hard-to-understand areas
~- [ ] I have made corresponding changes to the documentation~
- [x] I have made corresponding change to the default configuration files
~- [ ] I have added tests that prove my fix is effective or that my feature works~
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## How to test this PR locally

Not yet possible until https://github.com/elastic/uptime/issues/276 is ready

### CODE REVIEW

**Comment by michalpristas** [comment]
location is fine but if you want to pack it into a resulting package `packages.yml` needs to be updated

File: dev-tools/packaging/packages.yml [hunk 1]
```diff
@@ -45,6 +45,10 @@ shared:
         source: 'elastic-agent.yml'
         mode: 0600
         config: true
+      /etc/{{.BeatName}}/capabilities.yml:
```
Comment (michalpristas) [inline dev-tools/packaging/packages.yml:48]: this needs to be added in multiple places. 
try searching for `'elastic-agent.yml'` there should be 5 occurences, you have 1 so far.

**Comment by ruflin** [comment]
I wonder if this is the right approach to follow. So far we don't have any capabilities file shipped by default and it is up to the user to create one if needed. No matter what the user configures in the capabilities.yml, synthetics/browser will not work so it should be hardcoded in the build. My understanding is the the only build that supports synthetics/browser is the complete Dockerfile. Instead of doing modifications to the capabilities, instead heartbeat should return an error if this input is sent down and explain to the user that it is not supported. It is not up to Elastic Agent to know if the input is working in a certain environment or not but heartbeat.

My proposal is to move this implementation to heartbeat and not touch capabilities.yml file as it is not designed for this use case.

**Comment by andrewvc** [comment]
Closing this based on Ruflin's concerns and the fact that we already do surface an error message in heartbeat interviews to run browser-based monitors if we are not running on an image with the synthetics dependencies.

**Pull request closed without merging**